app/controllers/admin/settings/campaigns_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Admin
+  module Settings
+    class CampaignsController < Admin::ApplicationController
+      def create
+        result = ::Campaigns::SettingsUpsert.call(settings_params)
+
+        if result.success?
+          Audit::Logger.log(:internal, current_user, params.dup)
+          redirect_to admin_config_path, notice: "Site configuration was successfully updated."
```

Comment:
I wonder if `Site configuration` is the correct phrasing, but we might want to revisit this once we landed on proper naming in the admin restructure. Definitely premature now

Reply:
I wonder that for a while and figured I'll just switch it to "Settings were successfully updated" at the end. But then we probably should also change the menu entry which currently says "Config".